src/layouts/HorizontalLayoutRenderer.tsx

```diff
@@ -0,0 +1,39 @@
+import { LayoutProps, GroupLayout } from "@jsonforms/core"
+import isEmpty from "lodash.isempty"
+import { AntDLayoutRenderer, AntDLayoutRendererProps } from "./LayoutRenderer"
+import { HorizontalLayout } from "../ui-schema"
+import { Form, Row } from "antd"
+
+export function HorizontalLayoutRenderer({
```

Comment:
I think renderer doesn't fit with our naming convention--we're naming other React components FooControl because they correspond to a Control type UISchema Element. So consistency here would mean this component is called `HorizontalLayout`, do you agree?

Reply:
Yes, and that's the same thing I suggested [here](https://github.com/great-expectations/jsonforms-antd-renderers/pull/26#discussion_r1518233927) for `GroupLayoutRenderer`. This would mean:
- `HorizontalLayoutRenderer` -> `HorizontalLayout`
- `VerticalLayoutRenderer` -> `VerticalLayout`
- `GroupLayoutRenderer` -> `GroupLayout`

I will change them all in this PR unless you disagree.